subprojects/scala/src/integTest/groovy/org/gradle/scala/compile/CachedScalaCompileIntegrationTest.groovy

```diff
@@ -39,11 +39,11 @@ class CachedScalaCompileIntegrationTest extends AbstractCachedCompileIntegration
             }
 
             dependencies {
-                compile group: 'org.scala-lang', name: 'scala-library', version: '2.11.8'
+                compile group: 'org.scala-lang', name: 'scala-library', version: '2.11.11'
```

Comment:
I think it's fine to not upgrade everywhere. But why do we upgrade at all in this PR then?